scripts/testnet/sovereignBridge/config/header-verifier.snippets.sh

```diff
@@ -48,6 +45,28 @@ upgradeHeaderVerifierContract() {
     printTxStatus ${OUTFILE}
 }
 
+registerBlsPubKeysInHeaderVerifier() {
+    echo "Setting BLS pub keys in Header Verifier contract on main chain..."
+    checkVariables HEADER_VERIFIER_ADDRESS || return
+
+    BLS_PUB_KEYS=$(python3 $SCRIPT_PATH/pyScripts/read_bls_keys.py)
+
+    local OUTFILE="${OUTFILE_PATH}/register-bls-keys.interaction.json"
```

Comment:
Is the contract really reading json value for bls keys?

Reply:
Yes, the script will read the BLS keys and make a transaction to register them